torch/_dynamo/backends/common.py

```diff
@@ -40,6 +40,8 @@ def _wrapped_bw_compiler(*args, **kwargs):
         kwargs["inference_compiler"] = (
             kwargs.get("inference_compiler") or kwargs["fw_compiler"]
         )
+        # input mutations are hardened enough that all of our debugging backends should use them
+        kwargs["keep_inference_input_mutations"] = True
```

Comment:
Hmm... I added this because I basically require this config to be turned on for graphs that have storage resizing in them, and I want the aot_eager backend to work with storage resizing.

I also think input mutations have been around long enough that this seems like a reasonable default for aot_eager and aot_eager_decomp_partition backends. But I can move this to a separate PR (and just change those two backends directly)

Reply:
This seems fine to me.  Maybe it should become the default.